### PR TITLE
Fixes #11390: Resolving Grammatical mistakes in first timers Issue_template.

### LINKED ISSUE #11390: Correct grammatical errors in `first-timers-only` issue template
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new** and looking through our general [help wanted](https://github.com/publiclab/plots2/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
<details>
  <summary>💡 <i>Learn how to claim</i> 🙋</summary>
  <h3>Claiming an issue</h3>
  <p>Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.</p>
  <p>And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.</p>
</details>

- [ ] 📝 **Update** the file [.github/ISSUE_TEMPLATE/--first-timers-only.md](https://github.com/publiclab/plots2/blob/main/.github/ISSUE_TEMPLATE/--first-timers-only.md) in the `plots2` repository (press the little pen Icon) and edit the line as shown below.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -13,11 +13,11 @@ This has been marked as a good candidate for becoming a [first-timers-only issue

 ****

-Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.
+Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before or those who haven't contributed to open source before**.

-If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!
+If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from underrepresented groups in free and open-source software!

-We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝
+We know that the process of creating a pull request is one of the biggest barriers for new contributors. This issue is for you 💝

 If you have contributed before, **consider leaving this one for someone new** and looking through our general [help wanted](https://github.com/publiclab/plots2/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) issues. Thanks!
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming the `first-timers-only` issue. We're happy to keep it assigned to you as long as you need it if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below, and [reach out in our chatroom](https://publiclab.org/chat)!

### PR DESCRIPTION
**Pull request by ojninja16**
Fixes #11390  

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by TildaDares** [comment]
Hi @ojninja16, can you add a link to the issue? Thanks!

> One thing that can help to get started is to make sure you've included a link back to the original issue you're solving, in the
> format fixes #0000 (for example)

File: .github/ISSUE_TEMPLATE/--first-timers-only.md [hunk 1]
```diff
@@ -53,7 +52,7 @@ $DIFF
 
 2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.
 
-- [ ] 🏁 **Done** Ask in comments for a review :)
+- [ ] 🏁 **Done** Ask in the comments for a review :)
```
Comment (TildaDares) [inline .github/ISSUE_TEMPLATE/--first-timers-only.md:55]: Thanks for adding this @ojninja16!

Comment (ojninja16) [reply .github/ISSUE_TEMPLATE/--first-timers-only.md:55]: @TildaDares I added the issue link above .. Is it ok now??

Comment (ojninja16) [reply .github/ISSUE_TEMPLATE/--first-timers-only.md:55]: > Thanks for adding this @ojninja16!

Thank you for giving me this opportunity

**Review by TildaDares — APPROVED** [review]
Amazing job @ojninja16. Thank you!!

**Comment by TildaDares** [comment]
Congratulations on merging your PR @ojninja16 🎉 🎉

**Comment by ojninja16** [comment]
> Congratulations on merging your PR @ojninja16 🎉 🎉

Thank you so much @TildaDares @jywarren for giving me this opportunity to learn as well as contribute .. I would continue to contribute as much as i can....Thanks for all your help and cooperation

**Comment by ojninja16** [comment]
> Congratulations on merging your PR @ojninja16 🎉 🎉

I would really like to contribute more. I have a sound knowledge of HTML, CSS(bootstrap), Vanilla JS(learning react currently). Do let me know in any way I can contribute @TildaDares ..Thanks

**Comment by TildaDares** [comment]
Hi @ojninja16, you can take a look at the issues in this project to check for anyone that might interest you. You can also go through the publiclab.org site and search for bugs that you can fix or would like for another newcomer to solve. I’ll also keep an eye out for interesting issues that match your skill set. Thank you!

**Comment by ojninja16** [comment]
Sure, thanks for help @TildaDares !! much appreciated....

**Comment by TildaDares** [comment]
Hi @ojninja16, looks like there are two issue templates for `first-timers-only` issues. Do you think you could make the same changes in this file https://github.com/publiclab/plots2/blob/main/.github/first-timers-issue-template.md or you could use it as an FTO for someone else?

**Comment by ojninja16** [comment]
Hi @TildaDares , sure!!No problem

**Comment by TildaDares** [comment]
Thank you @ojninja16!

**Pull request merged**